shell/app/modules/application/pages/release/release-list.tsx

```diff
@@ -99,22 +101,23 @@ const ReleaseList = () => {
   const releaseId = get(list, `[${chosenPos}].releaseId`) || '' as string;
   return (
     <div className="release-list-container">
-      <div className="release-list-page column-flex-box">
-        <IF check={!params.appId}>
-          <div className="search-group">
-            <Search
-              value={projectId}
-              className="mb12"
-              placeholder={i18n.t('search by project id')}
-              onChange={(e) => !isNaN(+e.target.value) && updater.projectId(e.target.value)}
-            />
-            <Search
-              value={applicationId}
-              className="mb12"
-              placeholder={i18n.t('search by application id')}
-              onChange={(e) => !isNaN(+e.target.value) && updater.applicationId(e.target.value)}
-            />
-          </div>
+      <div className="release-list-page v-flex-box">
+        <IF check={appDetail.isDeployingApp}>
+          <AppSelector 
+            projectId={`${projectId}`} 
+            className='mb8 mx16'
+            allowClear
+            onChange={(_appId: number) => {
+              update({
+                applicationId: _appId,
+                queryObj: {
```

Comment:
how about set type of `queryObj ` as `null |  { q?: string, branchName?: string }`